bindata/network/ovn-kubernetes/common/008-script-lib.yaml

```diff
@@ -589,6 +589,19 @@ data:
         ovn_v6_join_subnet_opt="--gateway-v6-join-subnet {{.V6JoinSubnet}}"
       fi
 
+{{- if not .OVNIPsecEnable }}
+      # flush out xfrm state and policy entries so that it cleans up
+      # entries created for ovs. Since it's cleaning up whole entries,
+      # it may cause slight interruption in case of external ipsec
+      # configuration.
+      echo "remove ovs ipsec state if present"
```

Comment:
another important issue with this, is that AFAIK, it'll create connectivity issues for workloads!

removing the state, forces ipsec off, but if the other node is not updated at exact the same moment, it'll still try to do ipsec (according to it's policy)

I dont think there's a good solution for this besides leaving ipsec on until next reboot